bootstraps/stage2/spack.in.yaml

```diff
@@ -0,0 +1,52 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - glibc %gcc@=13.2.0 arch=linux-spack-x86_64 ^linux-headers%gcc@12
+  - binutils %gcc@=13.2.0 arch=linux-spack-x86_64
+  - gcc@13.2.0 %gcc@=13.2.0 ~bootstrap ~graphite +binutils arch=linux-spack-x86_64
+  - gmp +stage1 arch=linux-spack-x86_64
```

Comment:
`gmp` would also be built with stage0 gcc, right? Maybe be explicit?

Reply:
Probably best yeah.